### ISSUE TITLE
Guide "BUILDING A NATIVE EXECUTABLE" shall define available floating tags

### ISSUE THREAD

**Issue by sonic-martin** [quarkusio/quarkus#17722]
## Description
The Guide "QUARKUS - BUILDING A NATIVE EXECUTABLE" in section "Creating a Linux executable without GraalVM installed" reccomend to use "Please note that the above command points to a floating tag. It is highly recommended to use the floating tag, so that your builder image remains up-to-date and secure."
But the available floating tags are not listed.
They are only listed in some issues in the quarkus-images repo (e.g. https://github.com/quarkusio/quarkus-images/pull/93).

## Implementation ideas
List available floating tags in the mentioned guide.

**Comment by geoand** [comment]
The problem would be keeping these update.

cc @cescoffier

**Comment by cescoffier** [comment]
At the moment we don't have a way to extract the set of floating tags, except a link to the quay UI (https://quay.io/repository/quarkus/ubi-quarkus-native-image?tab=tags). Would that be ok?

**Comment by geoand** [comment]
That sounds perfectly reasonable to me

**Comment by gsmet** [comment]
Hmmm, AFAICS, this sentence is about Mandrel and there is a link to the tags already. If we want to point to the GraalVM images, we need to rework things. But I'm not sure it makes a lot of sense given default settings point to the image that makes the most sense.

I will fix the attribute though.

**Comment by sonic-martin** [comment]
> At the moment we don't have a way to extract the set of floating tags, except a link to the quay UI (https://quay.io/repository/quarkus/ubi-quarkus-native-image?tab=tags). Would that be ok?

Yes, and maybe additional in the Guide the general definition ```major.minor﻿-java[8|11]``` as in https://github.com/quarkusio/quarkus-images/pull/93. At least that both information I used to determine what to use.

**Comment by cescoffier** [comment]
We don't provide Java 8 anymore, so it will be 11, and later 11|17.
